apinotes/Foundation.apinotes

```diff
@@ -1738,3 +1738,13 @@ Enumerators:
   SwiftName: regularExpression
 - Name: NSStringEnumerationByLines
   SwiftName: byLines
+SwiftVersions:
+- Version: 3
+  Protocols:
+  - Name: NSFastEnumeration
+    Methods:
+    - Selector: 'countByEnumeratingWithState:objects:count:'
+      MethodKind: Instance
+      Parameters:
+      - Position: 1
+        Nullability: U
```

Comment:
"U"nspecified, i.e. ImplicitlyUnwrappedOptional. Swift (and Clang) ignored the nullability-on-arrays syntax in 3.0GM, so for source compatibility this changes this parameter from _Nonnull back to _Null_unspecified in Swift 3 mode.